src/plugins/plugin.url.js

```diff
@@ -283,19 +286,23 @@ export class UrlPlugin {
         const hasPropertyKey = doesKeyExists(urlStrSplitSlashObj, schema.name);
         const hasDeprecatedKey = doesKeyExists(schema, 'deprecated_for') && hasPropertyKey;
 
-        if (hasDeprecatedKey)
-          return urlState[schema.deprecated_for] = urlStrSplitSlashObj[schema.name];
+        if (hasDeprecatedKey) {
+          urlState[schema.deprecated_for] = urlStrSplitSlashObj[schema.name];
+          return;
+        }
 
-        if (hasPropertyKey)
-          return urlState[schema.name] = urlStrSplitSlashObj[schema.name];
+        if (hasPropertyKey) {
+          urlState[schema.name] = urlStrSplitSlashObj[schema.name];
+          return;
+        }
```

Comment:
Put these changes in your other PR, and then rebase this ; that way this PR becomes just about the volume sorting stuff!